go.mod

```diff
@@ -1,41 +1,42 @@
 module github.com/mailgun/holster/v3
 
-go 1.12
+go 1.15
```

Comment:
https://github.com/mailgun/dockerworld/pull/67
this is holding for some time now, since you upgrade this here, we should merge it, right ?